From 6e6fc364c1783ec71c7ef234555f1d73396feb6a Mon Sep 17 00:00:00 2001
From: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
Date: Wed, 1 May 2013 14:27:53 +0200
Subject: [PATCH] Disable documentation

We don't have latex2man yet, we might consider adding it to the base
system. Until then we disable documentation.
---
 Makefile.am |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index d7278df..67f4c97 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -36,7 +36,7 @@ endif
 
 nodist_include_HEADERS = include/libunwind-common.h
 
-SUBDIRS = src tests doc
+SUBDIRS = src tests
 
 noinst_HEADERS = include/dwarf.h include/dwarf_i.h include/dwarf-eh.h	\
 	include/compiler.h include/libunwind_i.h include/mempool.h	\
-- 
1.7.10.4

